.github/workflows/style_docstr.yml

```diff
@@ -47,19 +47,17 @@ jobs:
       - name: Install pyvista and doctest requirements
         run: pip install . -r requirements_test.txt
 
-      - uses: awalsh128/cache-apt-pkgs-action@v1.1.1
```

Comment:
Do we not need the cache action anymore? Or is that somehow incompatible with or included in our setup-headless-display action?

Reply:
>incompatible with or included in our setup-headless-display action?

Not incompatible! But also not included. See https://github.com/pyvista/setup-headless-display-action/issues/3

Reply:
@banesullivan so why is it removed in this PR? Or do you mean that your follow-up will fix this once it's merged?

Reply:
> Or do you mean that your follow-up will fix this once it's merged?

I've removed it since the logic will be moved to the headless action.